clouddriver-artifacts/src/integration/resources/gitea_data/gitea/indexers/issues.queue/LOG

```diff
@@ -0,0 +1,34 @@
+=============== Jan 5, 2021 (UTC) ===============
```

Comment:
Do we actually need to include all of the files in the `gitea_data` directory? Seems like a bunch of autogenerated files. Could remove them and Gitea still function properly?

Reply:
The files are there because after you start an empty gitea instance we need the following for the tests:

* Create a user
* Create a personal access token for that user
* Upload a ssh key for that user

These are already preloaded by the generated files. I faced something similar for creating the test repo and found a gitea REST endpoint to create it dynamically, I'll check if it's possible to populate those credentials via REST, so we can start with an empty gitea.